### ISSUE TITLE
'less' meet error with netstat -h and arp -h but dig -h

### ISSUE THREAD

**Issue by Justsoos** [termux/termux-app#680]
`netstat -h | less` or `arp -h | less` and other program get error with any up and down keys or inside command whether from busybox or installed by pkg, but `dig -h | less`  `netstat -na | less`shows good feedback.
my termux ver 0.60, from Google play store

**Comment by Justsoos** [comment]
seems an ancient bug of less pipe with xnix, I tried on my desktop with `netstat --help | less` within both ubuntu and centos, u c, `netstat --help` is valid options, still get error.
netstat is not only bug one, but I think it is a good case.

**Comment by Grimler91** [comment]
Yeah, you're right, -h and --help should both be acceptable options when I read the help more carefully.
I don't know enough to tell if it's a bug or a "feature" but it clearly causes some annoyance as you've demonstrated

**Comment by xeffyr** [comment]
@Justsoos netstat prints help to stderr, so you can try `netstat -h 2>&1 | less`.
